source/math/FloatTransform3D.ooc

```diff
@@ -208,7 +208,7 @@ FloatTransform3D: cover {
 	}
 	transformAndProject: func ~FloatPoint2D (point: FloatPoint2D, focalLength: Float) -> FloatPoint2D {
 		transformedWorldPoint := this * FloatPoint3D new(point x, point y, focalLength)
-		this project(transformedWorldPoint, focalLength)
+		focalLength < 0.000000001f ? FloatPoint2D new(transformedWorldPoint x, transformedWorldPoint y) : this project(transformedWorldPoint, focalLength)
```

Comment:
L269: epsilon ::= static FLT_EPSILON in math.ooc
This should be a better choice.